.prettierrc

```diff
@@ -2,6 +2,5 @@
   "trailingComma": "all",
```

Comment:
`.prettierrc.json` is our predominant style across the repos, any compelling reason to change it here?

Reply:
Ah, that was a test because I was having prettier issues. It turned out to be unrelated (CRLF linting error on Windows). 965adf6